CommonTools/Utils/test/testExpressionParser.cc

```diff
@@ -264,10 +264,9 @@ void testExpressionParser::checkAll() {
       std::cout << "Check for leaks in the " << (b ? "Lazy" : "standard") << " string parser" << std::endl;
       expr.reset();
       reco::parser::expressionParser<pat::Muon>("triggerObjectMatchesByPath('HLT_Something').size()", expr, b);
-      double res = 0;
       for (size_t i = 0; i < 10 * 1000; ++i) {
         for (size_t j = 0; j < 100; ++j) {
-          res += expr->value(o);
+          expr->value(o);
           break;
         }
         break;
```

Comment:
Given this `break`, what is the purpose of the `outer` for loop?

Reply:
I was also confused by this may be @VinInn  can help here